src/builder.ts

```diff
@@ -92,6 +92,16 @@ async function extractI18nMergeBuilder(options: Options, context: BuilderContext
         return isXliffV2 ? toXlf2(output, outputOptions) : toXlf1(output, outputOptions);
     }
 
+    function filterUnits(unit: TranslationUnit): boolean {
+        if (options.includeIdsWithPrefix) {
```

Comment:
```suggestion
        if (options.includeIdsWithPrefix?.length) {
```
Otherwise the default `[]` would make this always true?